packages/core/src/locale/language-service.spec.ts

```diff
@@ -70,7 +94,7 @@ describe('LanguageService', () => {
 
             expect(langService.translate('test.days_text', { count: 1 })).toBe('1 den');
             expect(langService.translate('test.days_text', { count: 2 })).toBe('2 dny');
-            expect(langService.translate('test.days_text', { count: 1.5 })).toBe('1.5 dne');
+            expect(langService.translate('test.days_text', { count: 1.5 })).toBe('1,5 dne');
```

Comment:
Curious to know why we need this change? Shouldn't it stay `1.5`?

Reply:
@richapatel1510 Great question. `1.5` should actually formatted as `1,5` for the `cs` locale. This is how number should be displayed in Czech. Displaying the decimal symbol as `.` is actually a bug. It has been fixed since `messageformat@3.0.0` (See https://github.com/messageformat/messageformat/commit/20d6fdbc9a30e49aa0bbad3a7bcc07625cb9d008). The alternative library `intl-messageformat` behaves the same way as the newer version of `messageformat`.